packages/evm/src/utils/helpers.ts

```diff
@@ -137,6 +141,24 @@ export const toHex = (covertThis: string | number | BigNumber, padding: number):
   return utils.hexZeroPad(utils.hexlify(amount), padding);
 };
 
+/**
+ * Return the address transformed to hex for bitcoin deposits
+ *
+ * @category Helpers
+ * @param address  - bitcoin address
+ * @param addressLength - length of the address
+ * @returns {string}
+ */
+export const addressToHex = (address: string, addressLength: number): string => {
```

Comment:
I think you can use decoders like this https://www.npmjs.com/package/@cmdcode/tapscript#address-tool

doesn't btc js already have such utils

Reply:
I'm not aware they have such utils. Will check this but I would avoid adding more dependencies to the SDK